python/ray/rllib/agents/sac/__init__.py

```diff
@@ -0,0 +1,4 @@
+from ray.rllib.agents.sac.config import DEFAULT_CONFIG
+from ray.rllib.agents.sac.trainer import SACTrainer
+
+__all__ = ["DEFAULT_CONFIG", "SACTrainer"]
```

Comment:
How about `SACPolicyGraph`?

Reply:
Hmm. You're right. I forgot that that's also exposed at the top level. We should add that here.